### ISSUE TITLE
[libsystemd] Build error on x64-linux

### ISSUE THREAD

**Issue by Eli5723** [microsoft/vcpkg#36411]
Program python3 (jinja2) found: NO

../src/v254-832da4078b.clean/meson.build:1997:0: ERROR: /usr/bin/python3 is missing modules: jinja2

**Comment by StarGate-One** [comment]
Please see #30604 How to report a package build failure.

**Comment by awsl12581** [comment]
I think you should use **pip install jinja2** beforehand

> Program python3 (jinja2) found: NO
> 
> ../src/v254-832da4078b.clean/meson.build:1997:0: ERROR: /usr/bin/python3 is missing modules: jinja2

**Comment by MonicaLiu0311** [comment]
We don't have enough information to solve this issue, unfortunately; if you have any more information to help us solve this, please reopen!
